syllabus/7-Polkadot/2-shared-security-slides.md

```diff
@@ -334,6 +348,18 @@ The new state root is then committed to the relay chain so the process can repea
 
 ---
 
+## Execution Core Math
+
+- Assuming we have 1023 validators...
+- Assuming we have each core be initially validated by 3 validators...
+- We can provide 341 parallel execution cores on Polkadot.
+
+From the JAM paper:
+
+<img style="width: 600px" src="../../assets/img/7-Polkadot/341-cores.png" />
+
+---
+
```

Comment:
Honestly I'd drop that. Think that is going a bit too deep into parametrisation and literally next lecture I'll be back to talking about the active validators set being 300 in polkadot. Let's try and avoid things that might confuse students.

And the number of cores is still up for debate. We recently changed vals per core to 3 from 5 but now we are changing it back to 5 because of poor performance. 3 might be optimistic and is just something students don't need to know at this point.

Reply:
sure